recipes/rmm/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+import os
+import textwrap
+from pathlib import Path
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import export_conandata_patches, get, copy, replace_in_file
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.52.0"
+
+
+class RmmConan(ConanFile):
+    name = "rmm"
+    description = "RAPIDS Memory Manager"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rapidsai/rmm"
+    topics = ("cuda", "memory-management", "memory-allocation", "rapids", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        # Based partially on https://github.com/rapidsai/rmm/tree/v23.06.00#get-rmm-dependencies
+        return {
+            "Visual Studio": "15",
+            "msvc": "191",
+            "gcc": "9.3",
+            "clang": "8",
+            "apple-clang": "14.0",
+        }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def configure(self):
+        # Thrust needs to use CUDA as the backend for RMM.
+        # Disabled for ConanCenter CI.
+        # self.options["thrust"].backend = "cuda"
+        pass
```

Comment:
```suggestion
    def configure(self):
        # Thrust needs to use CUDA as the backend for RMM.
        # Disabled for ConanCenter CI.
        # self.options["thrust"].backend = "cuda"
        pass
```
Please, move it to validate, but as a warning message. `self.output.warning("...")`